### ISSUE TITLE
Can't create a backend using a backend instance.

### ISSUE THREAD

**Issue by withrocks** [reclosedev/requests-cache#61]
According to the documentation, I should be able to pass an instance of a custom backend to `install_cache` (http://requests-cache.readthedocs.org/en/latest/user_guide.html#persistence).

However, `install_cache` calls `backends.create_backend` which only seems to support the registered names, so a `ValueError` is thrown:

```
def create_backend(backend_name, cache_name, options):
    if backend_name is None:
        backend_name = _get_default_backend_name()
    try:
        return registry[backend_name](cache_name, **options)
    except KeyError:
        raise ValueError('Unsupported backend "%s" try one of: %s' %
                         (backend_name, ', '.join(registry.keys())))
```

Am I missing something?

**Comment by reclosedev** [comment]
Thanks! FIxed in 0.4.12.

**Comment by withrocks** [comment]
Excellent. Thanks!
